docs/source/en/model_doc/albert.mdx

```diff
@@ -56,13 +56,27 @@ Next sentence prediction is replaced by a sentence ordering prediction: in the i
 This model was contributed by [lysandre](https://huggingface.co/lysandre). This model jax version was contributed by
 [kamalkraj](https://huggingface.co/kamalkraj). The original code can be found [here](https://github.com/google-research/ALBERT).
 
-## Documentation resources
+## Resources
 
-- [Text classification task guide](../tasks/sequence_classification)
-- [Token classification task guide](../tasks/token_classification)
-- [Question answering task guide](../tasks/question_answering)
-- [Masked language modeling task guide](../tasks/masked_language_modeling)
-- [Multiple choice task guide](../tasks/multiple_choice)
+A list of official Hugging Face and community (indicated by 🌎) resources to help you get started with ALBERT. If you're interested in submitting a resource to be included here, please feel free to open a Pull Request and we'll review it! The resource should ideally demonstrate something new instead of duplicating an existing resource.
+
+<PipelineTag pipeline="fill-mask" />
+- [`AlbertForMaskedLM`] is supported by this [example script](https://github.com/huggingface/transformers/tree/main/examples/pytorch/language-modeling#robertabertdistilbert-and-masked-language-modeling)
+
+<PipelineTag pipeline="text-classification"/>
```

Comment:
Similar to the fill-mask section, can you also add the relevant script and notebook for text classification with PyTorch, TensorFlow and Flax? This same comment goes for the token classification section, and we're also missing a multiple choice and question answering section.  See the [BERT](https://github.com/huggingface/transformers/blob/003a0cf8cc4d78e47ef9debfb1e93a5c1197ca9a/docs/source/en/model_doc/bert.mdx?plain=1#L62) resources for an example (and in some cases, you can just copy it over)!